ste/xfer-anyToRemote.go

```diff
@@ -289,7 +290,24 @@ func epilogueWithCleanupSendToRemote(jptm IJobPartTransferMgr, s ISenderBase, si
 		}
 	}
 
-	s.Epilogue()
+	s.Epilogue() // Perform service-specific cleanup before jptm cleanup. Some services may actually require setup to make the file actually appear.
+
+	// In theory, due to the limits imposed in copy.go, this should never be true unless it's a S2S transfer.
+	if info.S2SDestLengthValidation {
+		if s2sc, isS2SCopier := s.(s2sCopier); isS2SCopier { // TODO: Implement this for upload and download?
```

Comment:
I'm ok with putting this code here since we might enable this check for upload (any to remote doesn't include download).